tests/integration/pre-suite/01_websphere_module_install.rb

```diff
@@ -0,0 +1,33 @@
+test_name 'FM-3804 - C94728 - Plug-in Sync Module from Master with Prerequisites Satisfied on Agent'
+
+# step 'Install Module via PMT'
+# stub_forge_on(master)
+
+#Currently the puppetlabs-ibm_installation_manager is not ready for testing, so using joshbeard-ibm_installation_manager
+# instead.
+on(master, puppet('module install joshbeard-ibm_installation_manager'))
+
+on(master, puppet('module install nanliu-staging'))
```

Comment:
You should ping third-party Forge modules to specific version so that future changes in the modules don't break the CI pipeline unexpectedly.